tests/kernel/common/src/atomic.c

```diff
@@ -12,6 +12,16 @@
  * @{
  */
 
+#ifdef __LP64__
+#define MAXBIT		64
+#define INITIAL		0x0f0f0f0f0f0f0f0fL
+#define NAND_RESULT	0xFFFFFFFFFFFFF0FFL
+#else
+#define MAXBIT		32
```

Comment:
Did you consider giving this different name, like NUMBITS. Because I'd consider for 32 bits, "max bit" to be 31.